controllers/profile/profile.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package profile
+
+import (
+	"context"
+	"time"
+
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	// default reconcile timeout
+	reconcileTimeout = 1 * time.Minute
+
+	longWait = 1 * time.Minute
+
+	errGetProfile = "cannot get profile"
+)
+
+// SeccompProfileAnnotation is the annotation on a ConfigMap that specifies its
+// intention to be treated as a seccomp profile.
+const SeccompProfileAnnotation = "seccomp.security.alpha.kubernetes.io/profile"
```

Comment:
Do we want to remove the alpha because it will be GA in 1.19?
```suggestion
const SeccompProfileAnnotation = "seccomp.security.kubernetes.io/profile"
```

Reply:
I think that's a fair point. 

In terms of feature gates and feature life cycle, we can hold off dealing with them until we get a MVP out.